subprojects/gtest.wrap

```diff
@@ -2,12 +2,8 @@
 directory = googletest-release-1.11.0
 source_url = https://github.com/google/googletest/archive/release-1.11.0.zip
 source_filename = gtest-1.11.0.zip
-source_hash = 94c634d499558a76fa649edb13721dce6e98fb1e7018dfaeba3cd7a083945e91
+source_hash = 353571c2440176ded91c2de6d6cd88ddd41401d14692ec1f99e35d013feda55a
 patch_directory = gtest
 
 [provide]
-gtest = gtest_dep
-gtest_main = gtest_main_dep
-gmock = gmock_dep
-gmock_main = gmock_main_dep
 
```

Comment:
What happened here?

Reply:
Sanity check didn't like them:
```
AssertionError: Lists differ: ['gmock', 'gmock_main', 'gtest', 'gtest_main'] != []

First list contains 4 additional elements.
First extra element 0:
'gmock'

- ['gmock', 'gmock_main', 'gtest', 'gtest_main']
+ []
```

Reply:
That is because you needed to add them to releases.json too.

Reply:
Oh, of course.

Also, seems like there may be a bug, since it passed the sanity check just by adding `[provide]`. Might want it to check for the specific dependencies it's supposed to provide.  Would also help with more specific error messages.